contrib/ci/minimal-site/site.conf

```diff
@@ -0,0 +1,161 @@
+-- This is an example site configuration for Gluon v2018.2+
```

Comment:
maybe a git url for a site.conf could be useful in the Jenkinsfile as well - this would make the whole setup much more flexible.

Reply:
At least an override with a remote repository would be nice, so it can easily be reused downstream, but for us it is helpful to keep a working site in sync with the rest of the project.

Reply:
sure.